pkg/api/api.go

```diff
@@ -41,3 +43,22 @@ type BlockVolumeDeleteReq struct {
 	UnlinkStorage bool `json:"unlink-storage"`
 	Force         bool `json:"force"`
 }
+
+// BlockVolumeListResponse represents gluster block volume
+// list resp
+type BlockVolumeListResponse struct {
+	Blocks []string `json:"blocks"`
+	Result string   `json:"RESULT"`
+}
+
+// BlockVolumeInfoResponse represents gluster block volume
+// info resp
+type BlockVolumeInfoResponse struct {
```

Comment:
add a comment explains each field in this struct, it will help other we are new to this.

Reply:
this is taken from gluster-block cli output.